tests/gradients/core/test_pulse_gradient.py

```diff
@@ -919,7 +919,7 @@ def test_constant_ry(self, dev_name, num_split_times, t):
 
         dev = qml.device(dev_name, wires=1)
         # Effective rotation parameter
-        p = params[0] * (delta_t := (T[-1] - T[0]))
+        p = params[0] * (delta_t := T[-1] - T[0])
```

Comment:
# Question
Was this recommended by Pylint? I would imagine that
```python
p = x := (y - z)
```
is less ambiguous than
```python
p = x := y - z
```
because it is not obvious that this is _not_ equal to
```python
p = (x := y) - z
```

Reply:
it was a pylint thing.